pandas/core/indexes/base.py

```diff
@@ -3757,8 +3757,32 @@ def drop(self, labels, errors='raise'):
             indexer = indexer[~mask]
         return self.delete(indexer)
 
-    @Appender(base._shared_docs['unique'] % _index_doc_kwargs)
-    def unique(self):
+    base._shared_docs['index_unique'] = (
+        """
+        Return unique values in the index. Uniques are returned in order
+        of appearance, this does NOT sort.
+
+        Parameters
+        ----------
+        level : int or str, optional, default None
+            only return values from specified level (for MultiIndex)
+
+            .. versionadded:: 0.21.0
+
+        Returns
+        -------
+        Index without duplicates
+
+        See Also
+        --------
+        unique
+        Series.unique
+        """)
+
+    @Appender(base._shared_docs['index_unique'] % _index_doc_kwargs)
```

Comment:
I don't think the `_index_doc_kwargs` are doing something here ?

Reply:
@No, you're right. But it was already the case, so I thought it was a choice of consistency ("you don't need to change this line if you change the docs making the interpolation meaningful"). Shall I remove it?